### DISCUSSION TITLE
InputMode="DateInputMode.DateTime"

### DISCUSSION THREAD

**Discussion by uneone** [Megabit/Blazorise#2004] [Q&A]
<DateEdit TValue="DateTime?" InputMode="DateInputMode.DateTime" />

TimePart does not show up when after adding InputMode="DateInputMode.DateTime"

**Comment by stsrki** [comment]
What provider do you use? And what version?

**Reply by uneone** [reply]
I intalled blazorise.bootstrap and blazorise.Icons.FontAwesome from Visual Studio Nuget PackageManger.
version: 0.9.2.5
https://www.nuget.org/packages/Blazorise.Bootstrap/0.9.2.5/license

**Reply by stsrki** [reply]
That's your problem. `InputMode="DateInputMode.DateTime"` was only added in version 0.9.3. You will need to upgrade.

https://blazorise.com/news/release-notes/093/#migration-

**Reply by uneone** [reply]
Thank you.
